### PR TITLE
Header and intro section

### PR DESCRIPTION
**Pull request by KaskMIL**
# Header, Intro section and Github workflows
Mobile version
## Header
**Add a header with nav and style it**
## Intro
**Add a main section with a description**
## Github workflows
**Add Linters to help to master good practices**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <header>
+        <h1>Tomas Milanesi</h1>
+
+        <nav>
+            <i class="bi bi-list"></i>
+        </nav>
+    </header>
+
+    <main>
+        <h2>Hey There.<br>I'm Tom</h2>
```
Comment (prolajumokeoni) [inline index.html:23]: - [ ] Kindly change h2 tag  to h1 tag, as this is the main title of the page

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @KaskMIL, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR and Description



## Required Changes ♻️
_Check the comments under the review._

 

 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KaskMIL** [comment]
Hi @prolajumokeoni,
Thanks for your feedback, 100% clear on what I need to change to improve the page.
Thanks!

**Review by chrystalme — APPROVED** [review]
# Hi @KaskMIL,
## Status: Approved ✅ 

Nice job on following the previous reviewer's comments and implementing them accordingly. 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it.


Good luck on your next project! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by KaskMIL** [comment]
Thanks @chrystalme  and @prolajumokeoni !

**Pull request merged**